Header header
string device_name
string linux_version
string cpu_model_name
string user_name
# System
string[] partitions
float32[] system_size
float32[] free_space
# RAM
float64 ram_size
float64 used_ram
float64 available_ram
# CPU
uint32 cpu_processor
float32[] frequency
float64 cpu_temp
float32[] core_temp
